### ISSUE TITLE
Server crash on startup on Windows

### ISSUE THREAD

**Issue by NicoNekoDev** [PaperMC/Paper#3034]
### Link/paste of stack trace
https://pastebin.com/spshXDE1

### Plugin list:
No need, as it doesn't even start. 😂

### Actions to reproduce (if known):
All I did was: I've placed the paper-135.jar inside a folder, I've created the batch with the folowing command `java -jar paper-135.jar <new line> pause` and double clicked the batch to start the server.

### Paper build number:
This server is running Paper version git-Paper-135 (MC: 1.15.2) (Implementing API version 1.15.2-R0.1-SNAPSHOT) (I'm running the same jar file on my live production server, on a linux machine).

Probably you might want this?:
### System details:
java version "1.8.0_241"
Java(TM) SE Runtime Environment (build 1.8.0_241-b07)
Java HotSpot(TM) 64-Bit Server VM (build 25.241-b07, mixed mode)

OS: Windows 10 Pro, version: 1909, OS build: 18363.720

**Comment by NicoNekoDev** [comment]
I've just seen that I'm one version behind, I've downloaded it, but the same result on paper-136.

**Comment by NicoNekoDev** [comment]
After some non-debugging things, I've started to think that Windows is trying to (somehow) block paper? Evidence: https://www.youtube.com/watch?v=NiCe3julEjU
I can't open with a compiler, it crashes... etc.

**Comment by electronicboy** [comment]
check your antivirus software, make sure that you're not running it in some folder tained by something like onedrive, etc; Something is preventing paperclip accessing a file that's bundled in it

**Comment by NicoNekoDev** [comment]
There are some bugs occuring on windows. After stopping Windows Defender (the only anti-virus I have), the issue still happened. I tried opening the jar file using Administrator Access, but it said that the file can't be accessed. I've searched for the error and got the answer on stackoverflow: https://stackoverflow.com/questions/11943948/what-causes-unable-to-access-jarfile-error
`I just placed it in a different folder and it worked.`
Thank you.
